prodtests/full-system-test/dpl-workflow.sh

```diff
@@ -410,6 +414,7 @@ fi
 if [[ ! -z $INPUT_DETECTOR_LIST ]]; then
   if [[ $CTFINPUT == 1 ]]; then
     GPU_INPUT=compressed-clusters-ctf
+    has_detectors_gpu TPC ITS && GPU_INPUT+=",its-clusters"
```

Comment:
why do you do this only for CTFINPUT=1, I think this should be global?
Simiar for the GPU_OUTPUT above. In principle, I'd set the ITS input and outputs in GPU_INPUT and GPU_OUTPUT at the same place to avoid confusion.